src/compiler/diagnosticMessages.json

```diff
@@ -1923,6 +1923,10 @@
         "category": "Error",
         "code": 2685
     },
+    "Identifier '{0}' must be imported from a module": {

```

Comment:
We don't have a good way right now to generate `{1}` in that message (since we'd basically have to reverse-engineer a module name that would successfully resolve from the containing file).

How about

> `Global identifier {0} cannot be used from a module. Did you mean to import this from its module?`